pkg/logs/message/origin.go

```diff
@@ -32,7 +32,7 @@ func NewOrigin(source *config.LogSource) *Origin {
 //
 // The returned slice must not be modified by the caller.
```

Comment:
I don't think this line is true anymore?

It may also be worth adding that this is the combination of tags provided via SetTags and the source tags and the source category (so that callers don't try to get `.Tags()`, modify, and pass the result to `.SetTags()`)